docker/scripts/enterpoint.sh

```diff
@@ -23,6 +22,31 @@ while [ 1 ]; do
     fi
 done
 
+if [[ "${SEAFILE_LOG_TO_STDOUT:-false}" == "true" ]]; then
+    log "Creating symbolic links inside /opt/seafile/logs..."
+
+    mkdir -p /opt/seafile/logs/slow_logs
+
+    ln -sf /dev/stdout /opt/seafile/logs/controller.log
+    ln -sf /dev/stdout /opt/seafile/logs/file_updates_sender.log
+    ln -sf /dev/stdout /opt/seafile/logs/fileserver.log
+    ln -sf /dev/stdout /opt/seafile/logs/fileserver-error.log
+    ln -sf /dev/stdout /opt/seafile/logs/index.log
+    ln -sf /dev/stdout /opt/seafile/logs/notification-server.log
+    ln -sf /dev/stdout /opt/seafile/logs/notification-server-error.log
+    ln -sf /dev/stdout /opt/seafile/logs/onlyoffice.log
+    ln -sf /dev/stdout /opt/seafile/logs/seafdav.log
+    ln -sf /dev/stdout /opt/seafile/logs/seafevents.log
+    ln -sf /dev/stdout /opt/seafile/logs/seafile.log
+    ln -sf /dev/stdout /opt/seafile/logs/seafile-background-tasks.log.log
```

Comment:
TODO

Reply:
Fixed with [7573092](https://github.com/datamate-rethink-it/seafile-release/pull/27/commits/7573092828dce36ba409ae6594235dcdaa12ce47)